syntax = "proto2";
package Qot_UpdateTicker;

import "Common.proto";
import "Qot_Common.proto";

message S2C
{
	required Qot_Common.Security security = 1; //股票
	repeated Qot_Common.Ticker tickerList = 2; //逐笔
}

message Response
{
	required int32 retType = 1 [default = -400]; //RetType,返回结果
	optional string retMsg = 2;
	optional int32 errCode = 3;
	
	optional S2C s2c = 4;
}
